<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt RA Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_username">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>10</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Web API key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_key">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>10</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_tracker">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Tracker Window</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="opt_bgcolour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Use background colour</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="opt_bgimage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Use background image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choosebgcol">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>130</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_icons">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show achievement icons</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_unlocks">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show recent achievement unlocks</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_progress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show achievement progress</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="choosefont">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set display font...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Tracker</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_gamename">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show game name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_rank">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show total points and ranking</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh timer (seconds):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_refreshtime">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>42</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toggle_tracker</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tracker_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosebgcol</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choosebgcol_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosefont</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choosefont_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_username</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>user_updated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_key</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>apikey_updated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refresh_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opt_bgcolour</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bgcolour_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opt_bgimage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bgimage_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_gamename</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showgame_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_progress</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showprogress_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_icons</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showicons_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_rank</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showrank_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_refreshtime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshtimer_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tracker_clicked()</slot>
  <slot>choosebgcol_clicked()</slot>
  <slot>choosefont_clicked()</slot>
  <slot>user_updated()</slot>
  <slot>apikey_updated()</slot>
  <slot>refresh_clicked()</slot>
  <slot>bgimage_selected()</slot>
  <slot>bgcolour_selected()</slot>
  <slot>showgame_clicked()</slot>
  <slot>showprogress_clicked()</slot>
  <slot>showicons_clicked()</slot>
  <slot>showrank_clicked()</slot>
  <slot>refreshtimer_changed()</slot>
 </slots>
</ui>
